package.json

```diff
@@ -48,8 +48,8 @@
     "tslint": "^5.12.1",
     "tslint-config-prettier": "^1.13.0",
     "tslint-immutable": "^5.1.0",
-    "typedoc": "^0.14.2",
-    "typescript": "~3.2",
+    "typedoc": "^0.15.0-0",
```

Comment:
Do we really want to add this prerelease version?

Reply:
I'd prefer not to but this is the only way we can upgrade TypeScript. Given that `typedoc` or its results are only used by maintainers, not by users of IOV-Core, I think it is okay to use that version as long as `yarn docs` does not fail.